docs/INSTALL.md

```diff
@@ -151,7 +151,12 @@ However, it is possible to use the WSL (Windows Subsystem for Linux) to run GUI
 
 #### Windows 11 (WSL)
 
-Since Windows 11 Build 22000 running GUI apps from the WSL is supported [out-of-the box](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps). After [building](#building-1) the app you should be able to [run](#running) it without any additional steps.
+To run Linux GUI apps, you'll need to install a driver matching your system (see below). This enables a virtual GPU allowing hardware accelerated OpenGL rendering.
```

Comment:
Does it make sense to link the corresponding MS article?

Reply:
I can do - would you prefer this as well as the current text or as a replacement?

Reply:
What do you think? IMO MS will update relevant info if necessary, while our copy may become outdated some day.

Reply:
I feel the same, the only reason I did it like this was that [those instructions](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) are a small section to a larger article, so I wasn't sure it would be that clear. 

That was probably a bit overboard, I'm happy to change it if you think otherwise.

Reply:
The article covers it in full, that's great. What about this?

```suggestion
To run Linux GUI apps, you'll need to [install a driver](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) matching your system. This enables a virtual GPU allowing hardware-accelerated OpenGL rendering.
```
And remove everything that is below?